katas/content/complex_arithmetic/complex_conjugate/Solution.qs

```diff
@@ -1,7 +1,7 @@
-namespace Kata {    
-    open Microsoft.Quantum.Math;    
+namespace Kata {
+    open Microsoft.Quantum.Math;
 
     operation ComplexConjugate(x : Complex) : Complex {
```

Comment:
This is showing up in my editor with the lint warning that it should be a function. That's not something that intended to be fixed here?

Reply:
We were just focusing on structs and namespaces, as lints are disabled for katas. That being said, the placeholder is a function, so there's no reason for the solution to be an operation. I've updated it

Reply:
Unnecessary operation lint is disabled in the katas because sometimes the actual solution is a operation and placeholder is not. Prolly, we can enable lint rule for solutions written in katas to account for some of these. 

However, there are certain anomalities that might happen and  might be hard to automate. Also, not sure if they are worth the time and effort.

- If somebody accidently mentions something as operation in Placeholder which can be function instead(can get info from solution file). Additionally, typically 2-3 tasks are added at a time in katas mostly, so most of these should be easy to catch manually(though I know its good to automate as much as possible)

- With regards to above point, I dont think we can enforce if the learner decides to change function to operation while implementing something.